app/frontend/src/components/admin/AdminPage.vue

```diff
@@ -30,5 +31,14 @@ export default {
       adminDashboardUrl: this.$config.adminDashboardUrl,
     };
   },
+  computed: {
+    ...mapGetters('form', ['isRTL']),
+  },
 };
 </script>
+<style lang="css" scoped>
+.dir-rtl {
+  direction: rtl !important;
+  text-align: right;
+}
```

Comment:
Would it be better to have `.dir-rtl` as a global style instead of scoped here?

Reply:
Sorry - to clarify: could the `.dir-rtl` style be defined in `styles.scss`, so that it doesn't have to be defined on every page that uses it?

Reply:
I tried it but did not work well